tasks.py

```diff
@@ -39,25 +39,51 @@
 _CHECK_HELP_DESC = "Only checks for needed changes without writing back. Exit with error code if changes needed."
 _EXCLUDE_HELP_DESC = "Exclude files or directories"
 _PATH_HELP_DESC = "Target path. (Default: .)"
+# https://www.pyinvoke.org/faq.html?highlight=pty#why-is-my-command-behaving-differently-under-invoke-versus-being-run-by-hand
+_PTY_HELP_DESC = "Whether or not to use a pseudo terminal"
 
 
 @invoke.task(
     help={
         "check": _CHECK_HELP_DESC,
         "exclude": _EXCLUDE_HELP_DESC,
         "path": _PATH_HELP_DESC,
+        "isort": "Use `isort` to sort packages. Default behavior.",
+        "ruff": (
+            "Use `ruff` instead of `isort` to sort imports."
+            " This will eventually become the default."
+        ),
+        "pty": _PTY_HELP_DESC,
     }
 )
 def sort(
-    ctx: Context, path: str = ".", check: bool = False, exclude: str | None = None
+    ctx: Context,
+    path: str = ".",
+    check: bool = False,
+    exclude: str | None = None,
+    ruff: bool = False,
+    isort: bool = False,  # isort is the current default
+    pty: bool = True,
```

Comment:
@NathanFarmer if you or anyone want's to try out the `ruff` based sorting.
`invoke sort --ruff`